Assets/Scripts/PlayerControls.cs

```diff
@@ -15,6 +15,7 @@ public class PlayerControls : MonoBehaviour
 	//public TextMeshProUGUI WinText;
 	public int score;
 	public GameObject coinPrefab;
+	public GameObject t;
```

Comment:
trying to figure out where the "t" is getting used. Can you let me know in the comment.